--- modules/core/src/ocl.cpp	2015-12-15 15:01:21.102583501 +0000
+++ modules/core/src/ocl.cpp	2015-12-15 15:01:31.930828841 +0000
@@ -5950,10 +5950,10 @@
         else
 #endif
         {
-            CV_SUPPRESS_DEPRECATED_START
+            //CV_SUPPRESS_DEPRECATED_START
             CV_Assert(!alias);  // This is an OpenCL 1.2 extension
             handle = clCreateImage2D(context, CL_MEM_READ_WRITE, &format, src.cols, src.rows, 0, NULL, &err);
-            CV_SUPPRESS_DEPRECATED_END
+            //CV_SUPPRESS_DEPRECATED_END
         }
         CV_OclDbgAssert(err == CL_SUCCESS);
 


